src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1PlacementRequestEmailServiceTest.kt

```diff
@@ -321,6 +322,10 @@ class Cas1PlacementRequestEmailServiceTest {
       .withApArea(
         ApAreaEntityFactory()
           .withName(AREA_NAME)
+          .produce(),
+      )
+      .withCruManagementArea(
+        Cas1CruManagementAreaEntityFactory()
           .withEmailAddress(apAreaEmail)
```

Comment:
can we rename this variable please e.g. cruManagementAreaEmail

Reply:
Yep, done.